app/views/editions/secondary_nav_tabs/confirm_unpublish.html.erb

```diff
@@ -0,0 +1,40 @@
+<% @edition = @resource %>
+<% content_for :title_context, @edition.title %>
+<% content_for :page_title, "Unpublish" %>
+<% content_for :title, "Unpublish" %>
+<div class="govuk-grid-row">
+  <% unless @edition.errors.empty? %>
+    <% content_for :error_summary do %>
+      <%= render("govuk_publishing_components/components/error_summary", {
+        id: "error-summary",
+        title: "There is a problem",
+        items: @edition.errors.map do |error|
+          {
+            text: error.message,
+            href: "##{error.attribute.to_s}",
+          }
+        end,
+      }) %>
+    <% end %>
+  <% end %>
+
+  <div class="govuk-grid-row">
```

Comment:
Looks like this row is nested inside another. Probably for this view we can put everything in a single row.